Microsoft.Toolkit.Uwp.UI.Animations/ApiInformationHelper.cs

```diff
@@ -0,0 +1,32 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Windows.Foundation.Metadata;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    internal class ApiInformationHelper
```

Comment:
We shouldn't encourage checking for a specific version IMO, devs should still check for existence of a specific API. I know it's hypocritical, but the reason why I'm using this is to avoid calling the ApiInformation API multiple times